Diff testing may fail with Mutability Exception

See example: https://github.com/hashgraph/hedera-state-validator/actions/runs/20639690829/job/59269674085 
```
Starting json export with current version in background
Starting json export with next version in background
Waiting for current version export to finish
java.lang.RuntimeException: com.swirlds.base.state.MutabilityException: This operation is not permitted on immutable leaves
	at com.hedera.statevalidation.SortedExportCommand.run(SortedExportCommand.java:78)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2045)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
	at picocli.CommandLine.execute(CommandLine.java:2174)
	at com.hedera.statevalidation.StateOperatorCommand.main(StateOperatorCommand.java:49)
Caused by: com.swirlds.base.state.MutabilityException: This operation is not permitted on immutable leaves
	at com.swirlds.virtualmap.internal.cache.VirtualNodeCache.throwIfLeafImmutable(VirtualNodeCache.java:1572)
	at com.swirlds.virtualmap.internal.cache.VirtualNodeCache.clearLeafPath(VirtualNodeCache.java:649)
	at com.swirlds.virtualmap.VirtualMap.add(VirtualMap.java:1478)
	at com.swirlds.virtualmap.VirtualMap.put(VirtualMap.java:661)
	at com.swirlds.virtualmap.VirtualMap.put(VirtualMap.java:636)
	at com.swirlds.state.merkle.disk.OnDiskWritableSingletonState.putIntoDataSource(OnDiskWritableSingletonState.java:64)
	at com.swirlds.state.spi.WritableSingletonStateBase.commit(WritableSingletonStateBase.java:81)
	at com.swirlds.state.merkle.VirtualMapState$MerkleWritableStates.lambda$commit$4(VirtualMapState.java:669)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.base/java.util.stream.SortedOps$SizedRefSortingSink.end(SortedOps.java:357)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:510)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at com.swirlds.state.merkle.VirtualMapState$MerkleWritableStates.commit(VirtualMapState.java:669)
	at com.hedera.node.app.state.merkle.MerkleSchemaRegistry.migrate(MerkleSchemaRegistry.java:213)
	at com.hedera.node.app.services.OrderedServiceMigrator.lambda$doMigrations$0(OrderedServiceMigrator.java:79)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at java.base/java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1116)
	at com.hedera.node.app.services.OrderedServiceMigrator.doMigrations(OrderedServiceMigrator.java:72)
	at com.hedera.statevalidation.util.StateUtils.initServiceMigrator(StateUtils.java:223)
	at com.hedera.statevalidation.util.StateUtils.getDeserializedSignedState(StateUtils.java:121)
	at com.hedera.statevalidation.SortedExportCommand.run(SortedExportCommand.java:75)
	... 9 more
```

--- a/hedera-state-validator/src/main/java/com/hedera/statevalidation/blockstream/BlockStreamRecoveryWorkflow.java
+++ b/hedera-state-validator/src/main/java/com/hedera/statevalidation/blockstream/BlockStreamRecoveryWorkflow.java
@@ -69,7 +69,8 @@ public class BlockStreamRecoveryWorkflow {
             throw new RuntimeException(e);
         }
 
-        final SignedState signedState = deserializedSignedState.reservedSignedState().get();
+        final SignedState signedState =
+                deserializedSignedState.reservedSignedState().get();
         // need to create copy of the loaded state to make it mutable
         final HashedReservedSignedState hashedSignedState =
                 copyInitialSignedState(signedState, PlatformContextHelper.getPlatformContext());

